src/main/scala/centaur/test/formulas/TestFormulas.scala

```diff
@@ -39,11 +29,13 @@ object TestFormulas {
   def runSuccessfulWorkflowAndVerifyMetadata(workflowDefinition: Workflow): Test[Unit] = for {
     w <- runSuccessfulWorkflow(workflowDefinition)
     _ <- validateMetadata(w, workflowDefinition)
+    _ <- validateDirectoryContentsCounts(workflowDefinition, w.id.id.toString)
```

Comment:
Do you really need to drill into the wf that much to get it to stringify properly?